lib/resources.js

```diff
@@ -627,6 +643,15 @@ define( [
 
    ///////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+   var $q;
+
+   function resolvedPromise() {
+      var $q = $q || ng.injector( [ 'ng' ] ).get( '$q' );
```

Comment:
Here, the caching of `$q` will be void because of `var` (I'll fix it), and the next three lines can simply be `return $q.when()`.
Otherwise the patch LGTM!